### PR TITLE
fix dropdown max-width and ellipsis

### PR DESCRIPTION
**Pull request by krlakwa**

### CODE REVIEW

File: src/alto-ui/Dropdown/Dropdown.scss [hunk 1]
```diff
@@ -43,13 +43,14 @@
...
 .Dropdown__wrapper-trigger {
+  max-width: inherit;
   &--clearable {
     position: relative;
   }
 }
 
 .Dropdown__trigger {
-  max-width: 100%;
+  max-width: inherit;
```
Comment (renaudtertrais) [inline src/alto-ui/Dropdown/Dropdown.scss]: remove the property instead no ?

File: src/alto-ui/Dropdown/components/DropdownItem/DropdownItem.scss [hunk 2]
```diff
@@ -35,6 +36,10 @@
     text-decoration: line-through;
     background-color: $white;
   }
+
+  &--more {
+    min-width: 2rem;
```
Comment (renaudtertrais) [inline src/alto-ui/Dropdown/components/DropdownItem/DropdownItem.scss]: `flex-shink: 0;` better no ?

File: src/alto-ui/Dropdown/Dropdown.scss (hunk 1 above)
Comment (krlakwa) [reply src/alto-ui/Dropdown/Dropdown.scss]: then the issue isn't solve and the very long content goes out of the Dropdown__trigger

File: src/alto-ui/Dropdown/components/DropdownItem/DropdownItem.scss (hunk 2 above)
Comment (krlakwa) [reply src/alto-ui/Dropdown/components/DropdownItem/DropdownItem.scss]: better, I'll change it

**Pull request merged**